### PR TITLE
storing details in db api

### PR DESCRIPTION
**Pull request by Anikkk**

### CODE REVIEW

File: src/app.py [hunk 1]
```diff
@@ -67,6 +69,32 @@ def update_user():
     except Exception as e:
         return jsonify({"error": str(e)}), 500
 
+@app.route('/update_details', methods=['POST'])
+def update_details():
+    data = request.json
+    print(data)
```
Comment (kaivan-s) [inline src/app.py:75]: Remove this statement. Instead add a log.

File: src/app.py [hunk 2]
```diff
@@ -67,6 +69,32 @@ def update_user():
...
+        return jsonify({"error": "Invalid input"}), 400
+    username = data.get('username')
+    email = data.get('email')
+    phone = data.get('phone')
+    company = data.get('company')
+    if not userid or not username or not phone or not company:
+        return jsonify({"error": "Invalid input. Please provide userid, username, email, phone, and company."}), 400
+
+    table = get_dynamodb_client_table()
+    success, error_message = update_client_details(table, userid, username,email, phone, company)
```
Comment (kaivan-s) [inline src/app.py:87]: This entire logic should be somewhere else, the main function of this method is to call the update_client_details method. 
Logic should be in that file itself. 

Keep the main file clean.

**Review by kaivan-s — CHANGES_REQUESTED** [review]
Added a few comments. Please check

File: src/app.py (hunk 2 above)
Comment (Anikkk) [reply src/app.py:87]: the entire logic has been implemented in user details.py file
Screenshot 2024-03-23 101933

File: src/app.py (hunk 1 above)
Comment (Anikkk) [reply src/app.py:75]: done

File: src/Database/user_details.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        return False, error_message
+
+
+def get_user_details(email, table):
+    try:
+        response = table.query(
+            KeyConditionExpression=boto3.dynamodb.conditions.Key('email').eq(email)
+        )
+        items = response.get('Items', [])
+        print(items)
```
Comment (kaivan-s) [inline src/Database/user_details.py:53]: Remove this.

**Review by kaivan-s — CHANGES_REQUESTED** [review]
Looks Good. Just remove a print statement and it should be fine.

File: src/Database/user_details.py (hunk 3 above)
Comment (Anikkk) [reply src/Database/user_details.py]: done

**Review by kaivan-s — APPROVED** [review]

**Pull request merged**